app/views/groups/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="text-center">
+  <header class="p-3 bg-primary text-white d-flex align-items-start justify-content-between">
+    <%= link_to groups_path do %>
+      <i class="bi bi-arrow-left fs-2 text-white"></i>
+    <% end %>
+  </header>
+
+  <div class="d-flex align-items-center justify-content-between bg-light border shadow-sm bg-bodycol-12 p-3">
+    <p class="text-secondary mb-0"><%= @group.name %></p>
+    <p class="fw-bold mb-0">$ <%= @group.expenses.sum('amount') %></p>
+  </div>
+  <p style="color: green"><%= notice %></p>
+
+  <main class="container">
+    <h3 class="text-center m-4">Transactions</h3>
+    <div id="entities">
+        <div class="text-secondary">
+          <% if @group.recent_transactions.length === 0 %>
```

Comment:
Good morning 
Thank you for the review...
please you need to check very I don't have any error here everything is working fine and good ,I
add new category and its added successfully with no error please, I will send you the screen short to see please do check project you review very well to avoid confusion

![Screenshot (152)](https://user-images.githubusercontent.com/92458236/202990082-f26432f1-2806-4d47-ba9e-1f10dc8492a7.png)
![Screenshot (153)](https://user-images.githubusercontent.com/92458236/202990120-d6cc20d2-335c-43e2-a93b-27dcaf10b33a.png)
![Screenshot (154)](https://user-images.githubusercontent.com/92458236/202990146-749eed9f-f690-469e-abd5-2767a550db28.png)
![Screenshot (156)](https://user-images.githubusercontent.com/92458236/202990178-e85ec0e8-672e-431b-a934-ad7d36d308bb.png)
![Screenshot (155)](https://user-images.githubusercontent.com/92458236/202990209-3b0b377b-20eb-4a14-be90-d7bbe7077a2d.png)